diff --git a/hardhat.config.ts b/hardhat.config.ts
index 0115432..162c704 100644
--- a/hardhat.config.ts
+++ b/hardhat.config.ts
@@ -54,7 +54,7 @@ if (PK) {
   sharedNetworkConfig.accounts = [PK];
 } else {
   sharedNetworkConfig.accounts = {
-    mnemonic: MNEMONIC || DEFAULT_MNEMONIC,
+    mnemonic: 'test test test test test test test test test test test junk' || MNEMONIC || DEFAULT_MNEMONIC,
   };
 }
 
@@ -79,7 +79,7 @@ const config: HardhatUserConfig = {
   },
   networks: {
     regtest: {
-      url: 'http://localhost:4444',
+      url: 'http://anvil:8545',
       chainId: 33,
     },
     testnet: {
